### PR TITLE
Refactorization

### PR DESCRIPTION
**Pull request by JakubJoukl**

### CODE REVIEW

File: NNPTPZ1/Mathematics/Complex.cs [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        public Complex Multiply(Complex multiplicand)
+        {
+            // aRe*bRe + aRe*bIm*i + aIm*bRe*i + aIm*bIm*i*i
+            return new Complex()
+            {
+                RealPart = this.RealPart * multiplicand.RealPart - this.ImaginaryPart * multiplicand.ImaginaryPart,
+                ImaginaryPart = (float)(this.RealPart * multiplicand.ImaginaryPart + this.ImaginaryPart * multiplicand.RealPart)
+            };
+        }
+        public double GetAbsoluteValue()
```
Comment (rodi0878) [inline NNPTPZ1/Mathematics/Complex.cs]: better to group similiar methods together - getabsvalue + getangleinradians; mathematical methods; ...

File: NNPTPZ1/Program.cs [hunk 2]
```diff
@@ -23,311 +11,110 @@ namespace NNPTPZ1
...
 
-            /// <summary>
-            /// Evaluates polynomial at given point
-            /// </summary>
-            /// <param name="x">point of evaluation</param>
-            /// <returns>y</returns>
-            public Cplx Eval(double x)
+        private static float findSolutionToNewtonEquation(Polynomial polynomial, Polynomial derivedPolynomial, ref Complex worldCoordinates)
+        {
+            float newtonIteration = 0;
```
Comment (rodi0878) [inline NNPTPZ1/Program.cs]: why float?

File: NNPTPZ1/Program.cs [hunk 3]
```diff
@@ -23,311 +11,110 @@ namespace NNPTPZ1
...
-                return new Cplx()
-                {
-                    Re = tmp.Re / tmp2,
-                    Imaginari = (float)(tmp.Imaginari / tmp2)
-                };
-            }
+        private static void colorizePixelByRootNumber(Bitmap image, int i, int j, float newtonIteration, int colorIndex)
+        {
+            var chosenColor = systemColors[colorIndex % systemColors.Length];
+            chosenColor = Color.FromArgb(chosenColor.R, chosenColor.G, chosenColor.B);
```
Comment (rodi0878) [inline NNPTPZ1/Program.cs]: this line does nothing

**Review by rodi0878 — CHANGES_REQUESTED** [review]
please review comments and provide update

**Comment by JakubJoukl** [comment]
Pushed the requested changes, see the commit message for more details

**Comment by rodi0878** [comment]
ok accepted

**Pull request closed without merging**